recipes/rapidjson/1.1.x/conanfile.py

```diff
@@ -24,3 +24,7 @@ def package(self):
 
     def package_id(self):
         self.info.header_only()
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "RapidJSON"
```

Comment:
Will this break current dependence that are using `rapidjson`? 

[The doc says](https://docs.conan.io/en/latest/reference/conanfile/attributes.html#cpp-info) 
> The alternate name

But it's not very clear if it replaces the default package name

Reply:
`grep -Irn rapidjson | grep -i requires` returns no matches.

Reply:
This might indeed break external recipes, but the cmake script of rapidjson also uses `RapidJSON` as cmake name.

Reply:
It's the external dependencies that I had in mind. I agree is should make the original of the project. 
I take it from your reply is does indeed replace the default.

Reply:
For reference:
https://github.com/Tencent/rapidjson/blob/2661a17c7eaede8c881e7455f5a66fd593ed8633/CMakeLists.txt#L21
https://github.com/Tencent/rapidjson/blob/master/CMakeLists.txt#L201-L204